libs/wingc/src/type_check/symbol_env.rs

```diff
@@ -27,6 +30,36 @@ pub struct SymbolEnv {
 	statement_idx: usize,
 }
 
+impl Display for SymbolEnv {
+	fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
Not critical for this PR, but would be very useful to add some colorization for this (we already have the colorized crate). I suspect it'll be much easier to look at with colors (which will be automatically emitted on non tty outputs, so you don't need to worry about that).